Wireframe/index.html

```diff
@@ -3,11 +3,54 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
+    <title>Git</title>
```

Comment:
One of the acceptance criteria is: "The page header includes a title and description."
I can see you've included a title here. Maybe you can have a think about how to include the description?